sections/collage.liquid

```diff
@@ -39,12 +41,45 @@
                   {% endif %}
                 >
                   {%- if block.settings.image != blank -%}
-                    {%- capture sizes -%}(min-width: {{ settings.page_width }}px) {% if section.blocks.size == 1 %}calc({{ settings.page_width }}px - 100px){% else %}{{ settings.page_width | minus: 100 | times: 0.67 | round }}px{% endif %}, (min-width: 750px){% if section.blocks.size == 1 %} calc(100vw - 100px){% else %} 500px{% endif %}, calc(100vw - 30px){%- endcapture -%}
-                    {{
-                      block.settings.image
-                      | image_url: width: 3000
-                      | image_tag: loading: 'lazy', sizes: sizes, widths: '550, 720, 990, 1100, 1500, 2200, 3000'
-                    }}
+                    {%- liquid
+                      if section.settings.desktop_layout == 'left'
+                        assign large_block = forloop.first
+                      else
+                        assign large_block = forloop.last
+                      endif
+
+                      assign grid_space_desktop = settings.spacing_grid_horizontal | divided_by: 2 | append: 'px'
+                      assign grid_space_mobile = settings.spacing_grid_horizontal | divided_by: 4 | append: 'px'
```

Comment:
I wonder if this is really necessary. It feel like it's just a few pixels 🤔 Also it might not always be true as you could have a collage section with one block only, so none of that space is actually impacting the size of the image. 

At most it's going to be `20px`.